### PR TITLE
[GAL-2018] Speakers and schedule

### PR DESCRIPTION
**Pull request by electrofelix**
Add speakers and provisional program

### CODE REVIEW

File: content/events/2018-galway/speakers/diarmuid-ohara.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ linktitle = "diarmuid-ohara"
 
 +++
 
-Diarmuid is a Software Development Engineer with a background in Automated and Performance Testing.  In the last number of years he has been involved in the transformation of the RSPEAKERBIOD organisation in HPE from monolithic waterfall model to an agile model using DevOps and Continuous Delivery.  He has been instrumental in the creation of a Continuous Delivery Pipeline for HPE Support Center and is constantly looking for ways to improve it.
```
Comment (electrofelix) [inline content/events/2018-galway/speakers/diarmuid-ohara.md]: Thanks, missed that.

File: content/events/2018-galway/speakers/debbie-levitt.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ linktitle = "debbie-levitt"
...
+Debbie Levitt, CEO of Ptype UX & Product Design Agency, has been a UX strategist, designer, and trainer since the 1990s. As a “serial contractor” who lived in the Bay Area for most of this decade, Debbie has influenced interfaces at Sony, Wells Fargo, Constant Contact, Macys.com, Oracle, Cisco, and a variety of Silicon Valley startups. Clients have given her the nickname, “Mary Poppins,” because she flies in, improves everything she can, sings a few songs, and flies away to her next adventure.  Debbie has presented at conferences including eBay’s Developer Conference, PayPal’s Developer Conference, UXPA, and WeAreDevelopers. She is an O’Reilly published author and one of few instructors on the planet recommended by Axure. Outside of UX work, and sometimes during UX work, Debbie enjoys singing symphonic prog goth metal, opera, and New Wave. She’s now a Digital Nomad splitting her time between the USA and rural Italy.
```
Comment (electrofelix) [inline content/events/2018-galway/speakers/debbie-levitt.md]: Did I miss something in this file as well?

Comment (bridgetkromhout) [reply content/events/2018-galway/speakers/debbie-levitt.md]: It's nothing you missed - it's just a flaw in my janky bash that doesn't handle `&` well.

Comment (electrofelix) [reply content/events/2018-galway/speakers/debbie-levitt.md]: Interesting, in the other commit it actually highlighted the different text showing the `SPEAKERBIO` got replaced with `&` but for some reason it's not showing me that here but now I know what to look for I'm seeing it, I'll keep a eye out for it next time ;-)

**Pull request merged**